src/page-response-impact/rtStatistics.tsx

```diff
@@ -0,0 +1,42 @@
+import { RtRecord } from "../infection-model/rt";
+
+export function numFacilitiesWithRtLessThan1(
+  facilitiesRtRecords: RtRecord[][],
+) {
+  const latestRtValues = facilitiesRtRecords.map((rtRecords) => {
+    // Note: This assumes the first in the array is the latest because of the
+    // sorting on line 104 in infection-model/rt.tsx
+    return rtRecords[0].value;
```

Comment:
i'm pretty sure this is backwards? the last item should be the latest, i.e., they should be in chronological order

I myself have been wondering how safe it is to always assume they'll be in the right order (they currently _should be_ but this could change without warning and break various things) ... at the very least maybe adding a test on `cleanRtData` would make this a safer assumption